www/core/teststs/extension_sts.xml

```diff
@@ -27,15 +27,15 @@
 		<version>3.6.0-beta1</version>
 		<infourl title="Joomla!">https://www.joomla.org/announcements/release-news/5660-joomla-3-6-beta-1-released.html</infourl>
 		<downloads>
-			<downloadurl type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/3.6.0-alpha/Joomla_3.6.0-beta1-Beta-Update_Package.zip</downloadurl>
+			<downloadurl type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/3.6.0-beta1/Joomla_3.6.0-beta1-Beta-Update_Package.zip</downloadurl>
 		</downloads>
 		<tags>
 			<tag>stable</tag>
 		</tags>
 		<maintainer>Joomla! PLT</maintainer>
 		<maintainerurl>https://www.joomla.org</maintainerurl>
 		<section>STS</section>
-		<targetplatform name="joomla" version="3.[345]" />
+		<targetplatform name="joomla" version="3.[34567]" />
```

Comment:
> What happens, if we reach Joomla 3.10?

Then change the regex to be compatible with it or use a separate `<update>` definition for `3.1[0-9]`.